app/jobs/expire_old_ribbons_job.rb

```diff
@@ -0,0 +1,7 @@
+class ExpireOldRibbonsJob < ApplicationJob
+  queue_as :default
```

Comment:
We should probably use queues based on SLOs, https://alexis.bernard.io/blog/2023-10-15-background-job-queues-and-priorities-may-be-the-wrong-path.html is a half decent summary of doing so (and links off to Ben Sheldon of GoodJob maintainer, and Nate Berkopec of Speedshop … notoriety?)

Reply:
I've seen this in production at scale and agree.